examples/next/pages/ui/components/account-settings/delete-user-with-overrides/index.page.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Amplify, Auth } from 'aws-amplify';
+
+import {
+  Button,
+  Card,
+  Flex,
+  Heading,
+  AccountSettings,
+  Text,
+  TextField,
+  withAuthenticator, // IGNORE
```

Comment:
why `// IGNORE`?

Reply:
Oh, this is a plugin added from FileUploader docs so that docs doesn't show any snippets we don't want customers to see.

In this case, we wanted to hide Authenticator stuffs so that docs focus on account settings components. (And technically, Authenticator isn't required for these components).